test/object-store/migrations.cpp

```diff
@@ -3032,280 +2258,278 @@ TEST_CASE("migration: AdditiveDiscovered") {
          }},
     };
 
-    std::vector<SchemaMode> additive_modes = {SchemaMode::AdditiveDiscovered, SchemaMode::AdditiveExplicit};
```

Comment:
Reindenting makes this all marked as modified, but the actual change here is switching to use `GENERATOR()` for the two versions.